pkg/controller/controller_instance_test.go

```diff
@@ -115,42 +115,34 @@ func TestReconcileInstanceWithAuthError(t *testing.T) {
 
 	testController.reconcileInstance(instance)
 
+	// verify that one catalog client action occurred
 	actions := fakeCatalogClient.Actions()
-	assertNumberOfActions(t, actions, 1)
```

Comment:
This specific test must have slipped past me in an earlier test refactor; did the current methods not do something that you wanted?